fhir-install/Dockerfile

```diff
@@ -20,9 +20,9 @@ RUN unzip -qq /tmp/fhir-server-distribution.zip -d /tmp && \
 
 # ----------------------------------------------------------------------------
 
-FROM openliberty/open-liberty:20.0.0.9-full-java8-openj9-ubi
+FROM openliberty/open-liberty:20.0.0.12-full-java8-openj9-ubi
 
-ARG FHIR_VERSION=4.5.1
+ARG FHIR_VERSION=4.6.0
```

Comment:
Our build scripts must overwrite this value because obviously 4.5.1 wasn't right, but I thought it would still be good to try keeping this default value up-to-date

Reply:
yes they do